tests/e2e/lib/blocks/eventbrite.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import { waitForSelector, waitAndClick, waitAndType } from '../page-helper';
+
+export default class EventbriteBlock {
+	constructor( block, page ) {
+		this.blockTitle = EventbriteBlock.title();
+		this.block = block;
+		this.page = page;
+		this.blockSelector = '#block-' + block.clientId;
+	}
+
+	static name() {
+		return 'eventbrite';
+	}
+
+	static title() {
+		return 'Eventbrite';
+	}
+
+	static eventId() {
```

Comment:
It seems that this is a content ID. I would recommend moving it away from block class into the test itself.

Reply:
Yeh, good point, moving to the test makes this more flexible. The only issue I had was then getting it back into the static `isRendered` method, as the calling `isRenderedBlockPresent` only has class parameter - have added an extra args param to pass through any additional test specific data to this method, but don't know if this the best way to do this or not, let me know if I barking up the wrong tree here.